### PR TITLE
Draft: Aroma guide

### LINKED ISSUE #144: Add guide for installing Aroma
Since Aroma finally got a public beta, I believe an installation guide should be added, because Aroma adds many, _many_ useful features to the Wii U. 
Given that it is still a public beta, I personally think it should not be a part of the main guide just yet, and should be peppered with CYA warnings to let users know that while this is an option you can take, it's not wise to do so just yet.

### PR DESCRIPTION
**Pull request by Luca06Luwa**
This PR also covers some other issues and pull requests as well.
The list of things this PR will cover is:
#112 
#140 
#149 
#155 

Closes #144 

Note that this is a draft and should only be merged when aroma goes into stable and all parts that are still legacy hb are converted to aroma.

I've also allowed maintainers to modify this PR should any errors appear.

### CODE REVIEW

**Comment by Luca06Luwa** [comment]
Basic rundown of what's being merged in this PR.

1. The aroma guide replacing the tiramisu guide as the default guide.
2. Tiramisu guide moved into the previous methods page incase of legacy hb not working in aroma.
3. Added a faq relating to running .elf hb on aroma.
4. Merged changes from some issues that help the guide be a bit more informative.
5. Changed the apps from what was currently there to the aroma versions where possible. (hb-appstore still not ported yet)

TL;DR the guide is now ready for when aroma goes stable, all that needs to be done now is check for any mistakes.

**Comment by Luca06Luwa** [comment]
If this PR says that there are conflicts that have to be resolved, they can be ignored. Like I mentioned before, this is a complete redo of the guide for aroma, so the tiramisu guide isn't in the user-guide folder anymore.

**Comment by GaryOderNichts** [comment]
Hey, thanks for this draft.
We don't plan on fully replacing the Tiramisu guide with Aroma for now though.
Instead we've been working on making the Aroma environment installation an optional step after installing Tiramisu (See #159).
Feel free to leave feedback or proposals over there.
